hudi-spark/src/main/java/org/apache/hudi/DataSourceUtils.java

```diff
@@ -241,6 +241,13 @@ public static HoodieRecord createHoodieRecord(GenericRecord gr, Comparable order
     return new HoodieRecord<>(hKey, payload);
   }
 
+  /**
+   * Drop duplicate records from incoming records.
```

Comment:
actually, my bad. we should word it differently. "Drop records already present in the dataset". Drop duplicates might give a wrong notion.

Reply:
@nsivabalan Done, thanks!